site/e2e/helpers.ts

```diff
@@ -43,6 +44,15 @@ export function requiresEnterpriseLicense() {
   test.skip(!enterpriseLicense);
 }
 
+// requiresTerraform by default is enabled.
+export function requiresTerraform() {
```

Comment:
```suggestion
// requiresProvisioner by default is enabled.
export function requiresProvisioner() {
```

wdyt? it's not really _terraform_ that the tests depend on. they depend on a real provisioner, which in turn depends on terraform (and docker).

Reply:
The `echo` provisioner is a real provisioner in the sense it is a provisioner daemon running. I was thinking this is specifically a terraform provisioner.

Reply:
maybe `requireTerraformProvisioner` then?